specification/schemas.json

```diff
@@ -332,6 +332,9 @@
   "ServiceRate": {
     "$ref": "./schemas/ServiceRate.json"
   },
+  "ServiceRateImport": {
```

Comment:
The resource that it imports are ServiceRates though. Even the prices of the options are stored on the ServiceRate resource. Wouldn't it be more clear to know that ServiceRateImport results in ServiceRates instead of RateImport retulting in ServiceRates?

Reply:
I think of this schema as somewhat isolated from the service rates actually. The resource itself as represented in the api specification has no relation to service rates, it wrap an importable file, hence I am suggesting a simpler name for the users. Essentially, people know what rates are. The question "how can I import my rates" has a simple answer "through rates-import api". 

I just do not see practically how having "Service" as a prefix makes it more clear, I personally prefer a shorter, simpler and direct name.